src/docs/reference/scale-to-zero.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Scale to Zero
+---
+
+Scale to Zero means reducing unnecessary costs by ensuring you only use what you need, when you need it.
+
+## What is App Sleeping?
+
+App sleeping enables automated detection of an inactive [service](/develop/services) based on outbound traffic.
+
+Enabling App Sleep on a service tells Railway to stop the service when it is inactive, effectively reducing the overall cost to run it.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1696017787/events_gjkaob.png"
+alt="App Sleeping in the Activity Feed"
+layout="intrinsic"
+width={700} height={460} quality={100} />
+
+### Inactive Service Detection
+
+Inactivity is based on outbound packets. If no packets are sent from the service for over 10 minutes or longer, the service is considered inactive.
+
+
+### Reactivation
+
+When the service receives traffic from the _public network*_, it is sent a signal to wake.  
+
+Only when the service is ready, is traffic routed to it, ensuring no requests are dropped during the startup process.
+
+_*Currently, requests made to the service over the private network do not cause it to awake from sleep._
```

Comment:
Suggestion:
```suggestion
We wake a service when it sends or receives traffic to/from the internet.

 The first request made to or sent from a service wakes a sleeping service up. This may take a small amount of time for the service to spin up again on the first request (commonly known as "cold boot time").
```